include/game/bg_effects.h

```diff
@@ -25,20 +24,20 @@ struct UNK_3005B80_UNK4 {
 
     u8 unk13;
 
-    u16 unk14[16];
+    u16 palBuffer[32 / sizeof(u16)];
```

Comment:
I think a direct `16` is more helpful than the `32 / sizeof(u16)` since it's the regular palette size used everywhere.
We could alternatively add a constant like `DEFAULT_PALETTE_COLOR_COUNT`... maybe something smaller than that! ^^